.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+image:
+  file: .gitpod.Dockerfile
+tasks:
+ - name: Setup environment variables for ESP-IDF
+   command: |
+     source /home/gitpod/.espressif/frameworks/esp-idf-v4.4/export.sh > /dev/null 2>&1
+vscode:
+  extensions:
+    - matklad.rust-analyzer
```

Comment:
Gitpod uses the [Open VSX](https://open-vsx.org/) registry, where [rust-analyzer is still under `matklad.rust-analyzer`](https://open-vsx.org/extension/matklad/rust-analyzer), I will try to keep an eye on it in case it gets updated!

Reply:
Good to know.